lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -240,7 +241,7 @@ def _setup_mock_request(self, mock_request, include_depth=False):
             data["depth"] = 0
         self._set_mock_request_data(mock_request, data)
 
-    def create_thread_helper(self, mock_request, extra_data=None):
+    def create_thread_helper(self, mock_request, extra_data=None, extra_expected_data=None):
```

Comment:
Can we name them better? Perhaps extra_request_data and extra_expected_server_data?